.github/workflows/stage-build.yml

```diff
@@ -361,6 +361,14 @@ jobs:
             wait $pid
           done
 
+      - name: Update AI Help index
```

Comment:
Should this be in the prod yaml as well?

Reply:
No, we currently have a single Supabase project, and we can split it up later.